scapy/contrib/rtcp.py

```diff
@@ -105,7 +137,18 @@ class RTCP(Packet):
         # HEADER
         BitField('version', 2, 2),
         BitField('padding', 0, 1),
-        BitFieldLenField('count', 0, 5, count_of='report_blocks'),
+        ConditionalField(
+            BitFieldLenField('count', 0, 5),
+            lambda pkt: pkt.packet_type not in (205, 206)
+        ),
+        ConditionalField(
+            BitEnumField('tl_feedback_message_type', 0, 5, _transport_layer_feedback_messages),
+            lambda pkt: pkt.packet_type == 205
+        ),
+        ConditionalField(
+            BitEnumField('ps_feedback_message_type', 0, 5, _payload_specific_feedback_messages),
+            lambda pkt: pkt.packet_type == 206
+        ),
         ByteEnumField('packet_type', 0, _rtcp_packet_types),
```

Comment:
(value here)

Reply:
```
>>> bin(0x81)
'0b10000001'
```
> 0b10 (=2) -> version
> 0b0 (=0) -> padding
> 0b0 (=0) -> ps_feedback_message_type

Isn't it actually:
0b00001 (=0) -> ps_feedback_message_type ?

Reply:
Oh I missed you set the length to 5. It is still not going to work because of the other issue: when you are dissecting this part of the packet, packet_type hasn't been read yet. You should probably just use a "generic" field of 5 bits and use it in all cases, regardless of its name.

Reply:
Ok, thanks! I'll use some generic name for now.
Is there any way I could change the field name after dissection?

Reply:
That's not possible natively as far as I know :/ You could probably work it out with custom fields but I would argue it's not worth the trouble